app/src/pages/trace/TracePage.tsx

```diff
@@ -572,6 +579,98 @@ function RetrieverSpanInfo(props: {
   );
 }
 
+function RerankingSpanInfo(props: {
+  span: Span;
+  spanAttributes: AttributeObject;
+}) {
+  const { spanAttributes } = props;
+  const rerankingAttributes = useMemo<AttributeObject | null>(() => {
+    const rerankingAttrs = spanAttributes[SemanticAttributePrefixes.reranking];
+    if (typeof rerankingAttrs === "object") {
+      return rerankingAttrs as AttributeObject;
+    }
+    return null;
+  }, [spanAttributes]);
+  const query = useMemo<string>(() => {
+    if (rerankingAttributes == null) {
+      return "";
+    }
+    return (rerankingAttributes[RerankingAttributePostfixes.query] ||
+      "") as string;
+  }, [rerankingAttributes]);
+  const input_documents = useMemo<AttributeDocument[]>(() => {
+    if (rerankingAttributes == null) {
+      return [];
+    }
+    return (rerankingAttributes[RerankingAttributePostfixes.input_documents] ||
+      []) as AttributeDocument[];
+  }, [rerankingAttributes]);
+  const output_documents = useMemo<AttributeDocument[]>(() => {
+    if (rerankingAttributes == null) {
+      return [];
+    }
+    return (rerankingAttributes[RerankingAttributePostfixes.output_documents] ||
+      []) as AttributeDocument[];
+  }, [rerankingAttributes]);
+
+  const numInputDocuments = input_documents.length;
+  const numOutputDocuments = output_documents.length;
+  return (
+    <Flex direction="column" gap="size-200">
+      <Card title="Query" {...defaultCardProps}>
+        <CodeBlock value={query} mimeType="text" />
+      </Card>
+      <Card
+        title={`Input Documents (${numInputDocuments})`}
+        {...defaultCardProps}
+        defaultOpen={false}
+      >
+        {
+          <ul
+            css={css`
+              padding: var(--ac-global-dimension-static-size-200);
+              display: flex;
+              flex-direction: column;
+              gap: var(--ac-global-dimension-static-size-200);
+            `}
+          >
+            {input_documents.map((document, idx) => {
+              return (
+                <li key={idx}>
+                  <DocumentItem document={document} />
+                </li>
+              );
+            })}
+          </ul>
+        }
+      </Card>
+      <Card
+        title={`Re-ranked Documents (${numOutputDocuments})`}
+        {...defaultCardProps}
```

Comment:
Same as above - rely on titleExtra and counter.

Reply:
I added it and it looks like this.

<img width="250" alt="Screenshot 2023-10-12 at 8 59 14 AM" src="https://github.com/Arize-ai/phoenix/assets/80478925/d31ee5a6-f7a3-42b9-b45d-f64cc74fb8bb">